packages/edit-site/src/components/app/index.js

```diff
@@ -30,9 +31,15 @@ function AppLayout() {
 
 export default function App() {
 	useRegisterSiteEditorRoutes();
-	const routes = useSelect( ( select ) => {
-		return unlock( select( editSiteStore ) ).getRoutes();
+	const { routes, currentTheme, themeSupports } = useSelect( ( select ) => {
+		const { getCurrentTheme, getThemeSupports } = select( coreStore );
+		return {
+			routes: unlock( select( editSiteStore ) ).getRoutes(),
+			currentTheme: getCurrentTheme(),
+			themeSupports: getThemeSupports(),
```

Comment:
I don't think we need both of these values. The `theme_supports` is available for `getCurrentTheme`, and the `getThemeSupports` just returns a subset of data. They share the same resolver.